backend/routes/documents.py

```
@@ -8,24 +8,94 @@
+    try:
+        for pdf_path in PDF_BASE_DIR.glob("*.pdf"):
+            filename = pdf_path.name
+            doc_id = pdf_path.stem  # Filename without .pdf
+
+            documents.append(DocumentInfo(
+                document_id=doc_id,
+                display_name=_format_display_name(filename),
+                filename=filename,
+                size_bytes=pdf_path.stat().st_size
+            ))
+    except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)